docs/reference-guides/README.md

```diff
@@ -18,7 +18,7 @@
 
 ## [Hooks Reference](/docs/reference-guides/filters/README.md)
```

Comment:
Should we also rename this to "Hooks and Filters Reference" now? Ripple effects... 😬 

 ```suggestion
## [Hooks and Filters Reference](/docs/reference-guides/filters/README.md)
```

Reply:
In theory, the concept "Hooks" would already include "Filters" (and "Actions") so I think the current two changes are enough for now for the purpose of this PR. 
We'll think later of a better way to name these Reference API pages